main.tex

```diff
@@ -341,18 +728,125 @@ \subsubsection*{Mento Reserve Safety Fund}
     \item Voting rights: No.
```

Comment:
should probably refresh the token allocation section a bit to be aligned with latest updates, i.e. no longer using hedgey

Reply:
done